src/constants/commands.ts

```diff
@@ -44,6 +44,12 @@ export const MENTION_EACH = {
       type: 3,
       require: false,
     },
+    {
+      name: "dev",
+      description: "want to tag them individually?",
```

Comment:
Can you  change the description as this is feature flag

Reply:
done